### ISSUE TITLE
(infinity) Extend the results of the paper: Beat them!

### ISSUE THREAD

**Issue by redouane-dziri** [redouane-dziri/deep-music-classification#10]
**List of ideas to have better results than the paper's with a similar setting**

- [x] Normalize input into [0, 1] before feeding to the network

- [x] Experiment with other similar neural network architectures. Ideas: 
   - relu instead of tanh
   - more conv filters of smaller size
   - more conv layers
   - dropout
   - max instead of average pooling

- [x] Instead of majority voting to get the class of a track, sum up the softmax of each piece and take the max of that

- [x] Train for less epochs

**Comment by arnaudstiegler** [comment]
I would add to this to check for RNN models for that (if it does not require too much work)

**Comment by redouane-dziri** [comment]
> I would add to this to check for RNN models for that (if it does not require too much work)

add it to the checklist!

**Comment by arnaudstiegler** [comment]
Results are in the `evalute_GLCM.ipynb`, the concatenated model is much better than the stacked one, especially on the track level
